mpqp/execution/providers/ibm.py

```diff
@@ -57,13 +59,16 @@ def run_ibm(job: Job) -> Result:
 
 
 @typechecked
-def compute_expectation_value(ibm_circuit: QuantumCircuit, job: Job) -> Result:
+def compute_expectation_value(
+    ibm_circuit: QuantumCircuit, job: Job, simulator: Optional["AerSimulator"]
```

Comment:
the simulator can't be in job.device ?

Reply:
I tried documenting when it is mandatory to have it but my documentation is probably wrong, you should review this change